odk1-src/collect-form-map.rst

```diff
@@ -0,0 +1,36 @@
+.. spelling::
+
+  unfinalized
+  basemap
+
+Viewing filled forms on a map
+================================
+
+.. versionadded:: 1.25
+
+  `ODK Collect v1.25.0 <https://github.com/opendatakit/collect/releases/tag/v1.25.0>`_
+
+If you are collecting data related to physical locations, you can see a map of all filled forms from the :doc:`Fill Blank Form <collect-filling-forms>` menu. Use the map view to see your progress, to identify filled forms with bad locations and to plan your travel to the next location to survey.
+
+.. note:: 
```

Comment:
Maybe I'm being too lasse fair about this but it feels to me like we don't need this level of detail around it only working in 1.25. I'd imagine at least one person will see this problem and ask about it in the forum but it feels like the sort of thing that will get asked whether it's in the docs or not. What do you think?

Reply:
I think this behavior needs to be documented somewhere as being intentional and because it's not necessarily totally intuitive what the fix is. That is, is it downloading the form with a particular version? Pressing a button somewhere? ...? I agree that it's not critical to have it in user docs but I can't really think of a better place for it to go. Because we have such a long tail of people staying on old versions, it's bound to come up years from now and I'm not sure I'd be able to remember the details!

Reply:
Fair enough. I'm convinced! I think a note is the right formatting a well if it's going in.